tests/all/winch.rs

```diff
@@ -0,0 +1,41 @@
+use anyhow::Result;
+use wasmtime::*;
+
+#[test]
+fn compiles_with_winch() -> Result<()> {
+    let mut c = Config::new();
+
+    // DOIT: once we remove winch as a default feature before push, update the features for this
+    // test somehow to include the winch feature.
+    c.strategy(Strategy::Winch);
+
+    let engine = Engine::new(&c)?;
+
+    // Winch only supports a very basic function signature for now while it's being developed.
```

Comment:
Add in more tests once the trampoline accepts more arguments. Ideally, we should be able to test that registers get populated, and stack gets populated if needed.